.github/workflows/json-yaml-validate.yml

```diff
@@ -0,0 +1,25 @@
+name: json-yaml-validate 
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+  workflow_dispatch:
+
+permissions:
+  contents: read
+  pull-requests: write # enable write permissions for pull request comments
```

Comment:
So we're granting write permission to an external repo at a tag ...?

pretty sure we already turned off write permissions for all actions at the repo level

Reply:
We can remove that `pull-requests: write` line and disable the Action's ability to "comment" back on PRs to lock things down a bit more. Can optionally pin the commit sha rather than using a tag as well.

I also did not notice the `make verify` command and if that is sufficient, then perhaps this PR is not needed. Just let me know and I can either leave it open or close it out 👍